packages/graph/hash_graph/lib/graph/src/store/postgres/knowledge/entity/mod.rs

```diff
@@ -303,8 +318,15 @@ impl<C: AsClient> PostgresStore<C> {
                             ),
                         });
 
+                        let left_entity_edition_id = left_entity.metadata().edition_id().clone();
+
+                        subgraph.vertices.knowledge_graph.insert(
+                            left_entity_edition_id,
+                            KnowledgeGraphVertex::Entity(left_entity),
+                        );
+
                         self.traverse_entity(
-                            left_entity.metadata().edition_id(),
+                            left_entity_edition_id,
```

Comment:
The code modification could have been even smaller if I just changed `traverse_entity` to take the actual record rather than the identifier, (it also would be very mildly faster as it'd save a lookup in the map), however I think we want to keep this general and based on identifiers. As we may be able to combine the various `traverse` methods (e.g. `traverse_entity_type`) and in those scenarios we may _not_ have the actual record, but just the URIs from inside the types.